docs/ja/trusted-solutions/cloud.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: ShotGrid in the Cloud
 pagename: cloud-index
-lang: ja
+lang: en
```

Comment:
```suggestion
lang: ja
```

Reply:
@pratik-sayanekar thanks for following up! These will need to be fixed before we merge, as I've had to fix them in the past after every merge they've been updated in. They cause our doc links to break, so we'll need to ensure they reflect the respective languages: `ja`, `zh_CN`, `ko`.